frontend/public/components/storage-class.tsx

```diff
@@ -38,120 +39,131 @@ const tableColumnClasses = [
   Kebab.columnClass,
 ];
 
-const StorageClassTableHeader = () => {
-  return [
-    {
-      title: 'Name',
-      sortField: 'metadata.name',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[0] },
-    },
-    {
-      title: 'Provisioner',
-      sortField: 'provisioner',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[1] },
-    },
-    {
-      title: (
-        <>
-          Reclaim <span className="hidden-sm">Policy</span>
-        </>
-      ),
-      sortField: 'reclaimPolicy',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[2] },
-    },
-    {
-      title: '',
-      props: { className: tableColumnClasses[3] },
-    },
-  ];
+const StorageClassDetails: React.FC<StorageClassDetailsProps> = ({ obj }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <div className="co-m-pane__body">
+        <SectionHeading text={t('public~StorageClass details')} />
+        <div className="row">
+          <div className="col-sm-6">
+            <ResourceSummary resource={obj}>
+              <DetailsItem label={t('public~Provisioner')} obj={obj} path="provisioner" />
+            </ResourceSummary>
+          </div>
+          <div className="col-sm-6">
+            <dl className="co-m-pane__details">
+              <DetailsItem label={t('public~Reclaim policy')} obj={obj} path="reclaimPolicy" />
+              <dt>{t('public~Default class')}</dt>
+              <dd>{isDefaultClass(obj).toString()}</dd>
+              <DetailsItem
+                label={t('public~Volume binding mode')}
+                obj={obj}
+                path="volumeBindingMode"
+              />
+            </dl>
+          </div>
+        </div>
+      </div>
+    </>
+  );
 };
-StorageClassTableHeader.displayName = 'StorageClassTableHeader';
 
-const StorageClassTableRow: RowFunction<StorageClassResourceKind> = ({
-  obj,
-  index,
-  key,
-  style,
-}) => {
+export const StorageClassList: React.FC = (props) => {
+  const { t } = useTranslation();
+  const StorageClassTableHeader = () => {
+    return [
+      {
+        title: t('public~Name'),
+        sortField: 'metadata.name',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[0] },
+      },
+      {
+        title: t('public~Provisioner'),
+        sortField: 'provisioner',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[1] },
+      },
+      {
+        title: (
+          <>
+            <Trans i18nKey="public~Reclaim">
+              Reclaim <span className="hidden-sm">Policy</span>
+            </Trans>
```

Comment:
I think we need to rely on Patternfly truncating long values here instead of trying to hide part of the message with `hidden-sm`. We don't know how long the text will be when translated. I would just make this `t(public~Reclaim policy)`.

Reply:
updated